acceptance-tests/helpers/apps/push.go

```diff
@@ -77,6 +80,34 @@ func WithName(name string) Option {
 		a.Name = name
 	}
 }
+func (a *App) CleanFileFromAppDir(filename string) {
+	err := os.Remove(fmt.Sprintf("%s/%s", a.dir, filename))
+	if err != nil {
+		panic(err)
+	}
+
+}
+func WithYAMLFile(filename string, contents map[string]interface{}) Option {
+	return func(a *App) {
+		//convert to yaml parsable by hil
+		for k, v := range contents {
+			valueBytes, err := json.Marshal(v)
+			if err != nil {
+				panic(err)
```

Comment:
suggestion(non-blocking): it's more common to call `ginkgo.Fail(message)` as it gives a better call stack in a test.